consistency/tool/clustering/word_extraction_test.json

```diff
@@ -18,5 +18,55 @@
       "parameters":  [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ]
    }
     
-  ]
+  ], 
+
+  "test2": [ 
```

Comment:
Can we make the test_ids more descriptive? Instead of "test1", "test2", ... Something like "simple-test", "empty-operations", etc.